web/themes/contrib/civictheme/includes/utilities.inc

```diff
@@ -88,8 +88,12 @@ function civictheme_media_get_variables(MediaInterface $media): ?array {
     return NULL;
   }
 
+  if (civictheme_get_theme_config_manager()->load('components.attachment.use_media_name', TRUE)) {
```

Comment:
@joshua-salsadigital 
as you can see yourself from the fact that you updated 2 tests for an attachment and for  publication card), this solution affects more than what it should.

We need to update the way this works:
1. Add file label to as a variable `media_name`.
2. Move the config retrieval to sit outside of this function and based on it's value use the appropriate returned variable (`$variable['name'] = $variable['media_name']`)
3. Add a new setting for publication card to choose the media name - same as for Attachment. Sorry, I missed this requirement as well - did not realise that we have more than one component that handles file names. The settings should be per-component.